ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-ipv4-labeled-unicast.cfg

```diff
@@ -0,0 +1,58 @@
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+!
+router bgp 65101
+   router-id 192.168.255.3
+   no bgp default ipv4-unicast
+   distance bgp 20 200 200
+   graceful-restart restart-time 300
+   graceful-restart
+   maximum-paths 2 ecmp 2
+   neighbor EVPN-OVERLAY-PEERS peer group
+   neighbor EVPN-OVERLAY-PEERS remote-as 65001
+   neighbor EVPN-OVERLAY-PEERS update-source Loopback0
+   neighbor EVPN-OVERLAY-PEERS bfd
+   neighbor EVPN-OVERLAY-PEERS rib-in pre-policy retain all
+   neighbor EVPN-OVERLAY-PEERS ebgp-multihop 3
+   neighbor EVPN-OVERLAY-PEERS password 7 q+VNViP5i4rVjW1cxFv2wA==
+   neighbor EVPN-OVERLAY-PEERS default-originate route-map RM-FOO always
+   neighbor EVPN-OVERLAY-PEERS send-community
+   neighbor EVPN-OVERLAY-PEERS maximum-routes 0
+   neighbor MLAG-IPv4-UNDERLAY-PEER peer group
+   neighbor MLAG-IPv4-UNDERLAY-PEER remote-as 65101
+   neighbor PG-BGP-LU peer group
+   neighbor PG-BGP-LU remote-as 65555
+   neighbor PG-BGP-LU update-source Loopback0
+   neighbor 192.168.255.1 peer group EVPN-OVERLAY-PEERS
+   neighbor 192.168.255.2 peer group EVPN-OVERLAY-PEERS
+   neighbor 198.51.100.3 peer group PG-BGP-LU
+   !
+   address-family ipv4 labeled-unicast
+      update wait-for-convergence
+      bgp missing-policy direction in action deny
+      bgp additional-paths receive
+      bgp additional-paths send ecmp
+      bgp next-hop-unchanged
+      neighbor PG-BGP-LU activate
+      neighbor 198.51.100.1 activate
+      neighbor 198.51.100.1 additional-paths receive
+      neighbor 198.51.100.1 rcf in RCF_TEST(ARGS)
+      neighbor 198.51.100.1 graceful-restart
+      neighbor 198.51.100.1 additional-paths send ecmp
+      neighbor 198.51.100.1 maximum-routes 0
```

Comment:
i dont think we need to add the support for maximum-routes under this address family as after hitting the command cli exit back to global router bgp
cc: @ClausHolbechArista @gmuloc

Reply:
If the command is not "staying" in this context, it is indeed not a valid command here and should not be included in the model.

Reply:
issue was the cli output - should have been `maximum-advertised-routes`